src/BaGet.Core/Search/DependentsResponse.cs

```diff
@@ -19,6 +19,36 @@ public class DependentsResponse
         /// The package IDs matched by the dependent query.
         /// </summary>
         [JsonPropertyName("data")]
-        public IReadOnlyList<string> Data { get; set; }
+        public IReadOnlyList<DependentResponse> Data { get; set; }
+    }
+
+    /// <summary>
+    /// A package that depends on the queried package.
+    /// </summary>
+    public class DependentResponse
```

Comment:
Let's call this class [`DependentResult`](https://github.com/loic-sharma/BaGet/blob/7c514bfe57a59b7ed086e37b73d61a8194306a20/src/BaGet.Protocol/Models/SearchResponse.cs#L11) to mirror the `SearchResponse`'s [`SearchResult`](https://github.com/loic-sharma/BaGet/blob/7c514bfe57a59b7ed086e37b73d61a8194306a20/src/BaGet.Protocol/Models/SearchResult.cs#L12).